roles/network_plugin/flannel/tasks/main.yml

```diff
@@ -9,3 +9,19 @@
   register: flannel_node_manifests
   when:
     - inventory_hostname in groups['kube-master']
+
+- name: Flannel | Set CNI directory permissions
+  file:
+    path: /opt/cni/bin
+    state: directory
+    owner: kube
+    recurse: true
+    mode: 0755
+  register: cni_bin_dir
+
+- name: Flannel | Copy CNI plugins
+  unarchive:
+    src: "{{ local_release_dir }}/cni-plugins-linux-{{ image_arch }}-{{ cni_version }}.tgz"
+    dest: "/opt/cni/bin"
+    mode: 0755
+    remote_src: yes
```

Comment:
cr need

Reply:
Sorry don't understand the 'cr' ?

Reply:
Line feed character
sorry for my slang

github show "no newline at end of file"